ansible/bootstrap.yml

```diff
@@ -86,31 +90,14 @@
 
     - name: Run the nomad acl bootstrap process
       include_tasks: nomad_acl_bootstrap.yml
-      when: "{{ lookup('env', 'TF_VAR_nomad_acl') }}"
+      when: lookup('env', 'nomad_acl') | bool
 
     - name: Create roles for nomad in vault
       include_tasks: vault_nomad_integration.yml
-      when: "{{ lookup('env', 'TF_VAR_nomad_acl') }}"
+      when: lookup('env', 'nomad_acl') | bool
 
     - name: Vault enable consul secret engine
       include_tasks: vault_consul_integration.yml
-      when: "'{{ lookup('env', 'consul_acl_default_policy') }}' == 'deny'"
```

Comment:
Shouldn't we include this check here:
when: lookup('env', 'consul_acl') | bool

Reply:
Yes. I do not know if this breaks anything, but will push an update.